Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -25,7 +25,14 @@ you need to make sure that the 'Gaze Input Capability' is checked on the 'Appx B
 
 This tooling will find the AppX manifest after the Unity build is completed and manually add the GazeInput capability.
 **Note that this tooling is NOT active when using Unity's built-in Build Window** (i.e. File -> Build Settings).
-When using Unity's build window, the capability will need to manually added after the Unity build.
+
+When using Unity's build window, the capability will need to manually added after the Unity build, as follows:
```

Comment:
"will need to be manually" - Add "be"

Reply:
To be, or not to be, that is the typo.
Now fixed!